xml/System.Text.Json.Serialization.Metadata/JsonTypeInfo.xml

```diff
@@ -20,15 +20,7 @@
   <Interfaces />
   <Docs>
     <summary>Provides JSON serialization-related metadata about a type.</summary>
-    <remarks>
-      <format type="text/markdown"><![CDATA[
-
-## Remarks
-
-This API is intended for use by the output of the System.Text.Json source generator and should not be called directly.
```

Comment:
Arguably, this statement is inaccurate even in .NET 6 -- `JsonTypeInfo` plays a prominent role in source generated APIs that users call into.

Reply:
These doc comments aren't versioned, so the remark will be removed for .NET 6 as well.